docs/specification.md

```diff
@@ -281,7 +283,7 @@ The "Merge Contributors Modal" includes:
 
 - “Current contributor” field (disabled): Prefilled with current contributor name and git email.
 - “Contributor to merge with” select: Has contributors list as options with their git emails in the parenthesis.
-- “Merge” button: When clicked, submits the form, closes the modal, and updates the contributor's data on the page. After merge, the selected contributor is deleted and his git emails are transferred to the current contributor.
+- “Merge” button: When clicked, submits the form, closes the modal, and updates the contributor's data on the page, the message "Contributors successfully merged" appears. After merge, the selected contributor is deleted and his git emails are transferred to the current contributor.
```

Comment:
Same
```suggestion
- “Merge” button: When clicked, submits the form, closes the modal, and updates the contributor's data on the page, the message "Contributors successfully merged." appears. After merge, the selected contributor is deleted and his git emails are transferred to the current contributor.
```

Reply:
I fixed in [commit](https://github.com/BinaryStudioAcademy/bsa-2024-gitfit/pull/96/commits/665b01927a40826cab3bf6d33e6d8697e91fa3d5)